examples/ConsoleApp/AccountServiceContext.cs

```diff
@@ -1,12 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
-using Consoto.Banking.AccountService.FeatureFilters;
-
-namespace Consoto.Banking.AccountService
```

Comment:
why'd the namespace go away?

Reply:
The ConsoleApp and other templates now defaults to https://aka.ms/new-console-template

Namespace automatically matches the project name. I left the namespace, "class Program", "Main" scaffolding in some of the Examples, but wanted to use the new console template in at least a few places to better match the new experience.